lib/data-types.js

```diff
@@ -38,12 +39,20 @@ var ABSTRACT = function(options) {
 
 };
 
+ABSTRACT.prototype.dialectTypes = '';
+
 ABSTRACT.prototype.toString = function() {
   return this.toSql();
 };
 ABSTRACT.prototype.toSql = function() {
   return this.key;
 };
+ABSTRACT.prototype.warn = function(text) {
+  if (!warnings[text]) {
+    warnings[text] = true;
+    console.warn('>> WARNING:', text, '\n>> Check:', this.dialectTypes);
```

Comment:
It's not really a deprecation, i'm not sure if depd explicitely uses that wording though to.